app/registries/menu/toggle.rb

```diff
@@ -1,7 +1,9 @@
 module Menu
   class Toggle < Node
-    def initialize(name, caption)
+    attr_accessor :icon
+    def initialize(name, caption, icon)
       @caption = caption
+      @icon = icon || ""
```

Comment:
@Rohoover in the rare case that a plugin did not define a menu icon, should we have a default icon or the default should be not to display any Icon?

Also, i saw in other projects that they use the ```>> ``` as a way to show there is a sub menu, should we use that? I personally found it noisy (for sure in the top level).

Reply:
I think a default icon would be good. I don't know if PatternFly defines what that icon is. 

In the demo and in my mockup, I incorporate the ">" for sub menus. I believe I commented earlier in the thread that it's missing. I think it's worth keeping because it differentiates between those items with and without a sub menu.

Reply:
@Rohoover So to sum it up:
1. scrollbar should be consistent across browsers? (Tbh that's sounds like a patternfly bug).
2. Missing › string on non links (including the top level navigation that are all links to sub menus)?

Anything else?

Reply:
@Rohoover 
All of the top level navigation items have a sub menu, so IMHO the ">" a bit noisy on them, especially in collapse mode.

Reply:
> scrollbar should be consistent across browsers? 

I don't think we should block this PR on the scrollbar issue for three reasons:

1. My experience with styling scrollbars consistently is that it's difficult (although perhaps CSS3 makes this easier)
1. Perhaps styling the scrollbar isn't even the correct solution here.  My point is that we should spend a little bit of time coming up with a correct solution instead of rushing into a solution.
1. You have to have a pretty small screen to see this issue.  I had to take my height down to around 600px to see it on the content menu (which is our longest menu)

> (Tbh that's sounds like a patternfly bug).

I would also be interested to know how patternfly as a whole plans on addressing this issue.

TL;DR: I would be fine with opening a follow up issue to address the scrollbar.

Reply:
Ping @rohoover I believe these are the only remaining issues before we can merge this?

Reply:
Yes, I believe so.

Reply:
@rohoover - I'm uncertain how to continue, the scrollbar seems like a pf bug, and I'm uncertain if we should block on it.
secondly, since we have no third level naviation, it seems that we don't need a ">" symbol? is that right?

what would you suggest we do ? thanks.

Reply:
I don't think we need to block on scroll bar, it's more a visual issue than functionality. As walden pointed out, it's probably an extreme use case that would even see it.

The chevron should be present on any menu that has a sub.

Reply:
thanks @Rohoover.
@amirfefer can you please
1. open a bug at pf for the scroll issue (that we track it).
2. add the missing chevrons ?

Reply:
@ohadlevy  what about a default icon? should we remain it blank for now?